README.md

```diff
@@ -48,6 +48,7 @@ You will need development headers/libraries for the following:
 * libhttp_parser [libhttp-parser-dev].
 * sqlite3 [libsqlite3-dev]
 * libolm [libolm-dev] (if not available, compile with `make MATRIX_NO_E2E=1`)
+* libgcrypt20 [libgcrypt20-dev] (if not available, compile with `make MATRIX_NO_E2E=1`)
```

Comment:
the library's just called `libgcrypt` afaik. `libgcrypt20` is a debian thing

I suggest:

```
libgcrypt [libgcrypt20-dev] (if not available, compile with `make MATRIX_NO_E2E=1`)
```

Reply:
OK- will make adjustments and push once more.